pyproject.toml

```diff
@@ -1,3 +1,86 @@
+[tool.poetry]
+name = "ledfx"
+version = "2.0.84"
+description = "A network based light effect controller"
+authors = ["Your Name <you@example.com>"]
+license = "MIT License"
+readme = "README.rst"
+packages = [
+	{include = "ledfx"},
+	{include = "ledfx_frontend"},
+    {include = "loopback"},
+]
+include = [
+    "README.rst",
+    "AUTHORS.rst",
+    "CHANGELOG.rst",
+    "LICENSE.txt",
+]
+
+[tool.poetry.scripts]
+ledfx = "ledfx.__main__:main"
+ledfx-loopback-install = "loopback.__main__:copy_lib"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.26"
+cffi = [
+    # Later than 1.15 on Linux fails.
+    {version = "<=1.15.0", platform = "linux"},
+    # Earlier than 1.16 on Windows Python 3.12 fails.
+    {version = ">=1.16.0", platform = "win32"},
+]
+wheel = ">=0.36.2"
+aiohttp = "^3.9.1"
+aiohttp-cors = ">=0.7.0"
+aubio = ">=0.4.9"
+cython = ">=0.29.21"
+certifi = ">=2020.12.5"
```

Comment:
What happened to all the package bumps ins the version I put in discord. Why do we have all these upper bounds?